plugins/amazonq/chat/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/FeatureDevSessionContextTest.kt

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.testFramework.RuleChain
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import software.aws.toolkits.jetbrains.services.amazonq.FeatureDevSessionContext
+import software.aws.toolkits.jetbrains.services.amazonqFeatureDev.FeatureDevTestBase
+import software.aws.toolkits.jetbrains.services.amazonqFeatureDev.util.FeatureDevService
+
+class FeatureDevSessionContextTest : FeatureDevTestBase() {
+    @Rule
+    @JvmField
+    val ruleChain = RuleChain(projectRule, disposableRule)
+    private lateinit var featureDevSessionContext: FeatureDevSessionContext
+    private lateinit var featureDevService: FeatureDevService
+    private lateinit var txtFile: VirtualFile
+    private lateinit var ktFile: VirtualFile
+    private lateinit var directory: VirtualFile
```

Comment:
these are all used in a single test so they dont really need to be initialized in setUp()